### ISSUE TITLE
swagger-ui-react: Module not found: Can't resolve '@babel/runtime-corejs2'...

### ISSUE THREAD

**Issue by robertying** [swagger-api/swagger-ui#5505]
### Q&A
 - Method of installation: yarn add swagger-ui-react
 - Swagger-UI version: 3.23.2

### Content & configuration

Clean create-react-app project

### Describe the bug you're encountering

Cannot compile because of the error with `@babel/runtime-corejs2` missing:

```
Failed to compile.

./node_modules/swagger-ui-react/index.js
Module not found: Can't resolve '@babel/runtime-corejs2/core-js/json/stringify' in 'C:\Users\username\project\node_modules\swagger-ui-react'
```

### To reproduce...

1. Create a new react project using create-react-app
2. `yarn add swagger-ui-react`
3. `import SwaggerUI from "swagger-ui-react";`
4. `yarn start`

### Expected behavior

It can compile with `swagger-ui-react` 3.23.0

### Additional context or thoughts

`@babel/runtime-corejs2` apparently is not used in create-react-app projects. So it should be declared as `dependencies` for `swagger-ui-react`. Adding it explicitly to `package.json` is not a common practice.

**Comment by shockey** [comment]
@robertying, we're pushing out a fix for this now!

**Comment by robertying** [comment]
@shockey Thanks! Good job 👍

**Comment by shockey** [comment]
FWIW, the smoke tests I've been building would've caught this, perhaps it's time to go prime time with them:

<img width="1139" alt="image" src="https://user-images.githubusercontent.com/680248/62154337-51a66a80-b2bb-11e9-9731-243aea8fb9d6.png">

**Comment by shockey** [comment]
## Fixed in [`swagger-ui-react@3.23.3`](https://unpkg.com/browse/swagger-ui-react@3.23.3/package.json).

Sorry for the inconvenience, @robertying! Appreciate the report.
